src/steps/users/converters.ts

```diff
@@ -106,6 +164,7 @@ export function createUserEntity(data: admin_directory_v1.Schema$User) {
         ...getEmails(data),
         ...getManagementInfo(data),
         ...getEmployeeInfo(data),
+        ...convertCustomSchemas(data.customSchemas),
```

Comment:
Should these properties all be prefixed with `customSchemas.`?
```suggestion
        ...convertCustomSchemas(data.customSchemas, 'customSchemas'),
```

I'm nervous that, for example, the following code would overwrite `userId` with `'some-other-id'`.

```ts
  const customSchemas = { id: 'some-other-id' };

  return createIntegrationEntity({
    entityData: {
      source: data,
      assign: {
        _key: getUserEntityKey(userId),
        _type: entities.USER._type,
        _class: entities.USER._class,
        id: userId,
        // omitting a number of properties
        ...convertCustomSchemas(data.customSchemas),
      },
    },
  });
```